pygazebo/pygazebo.cc

```diff
@@ -264,6 +265,24 @@ class Agent : public Model {
     }
   }
 
+  bool TakeActionPDController(const std::map<std::string, double>& ctrl_target) {
```

Comment:
we can merge this function with TakeAction(), adding another arguments: velocities

Reply:
Do we need postion controlled joints? For some sinario like rotating camera. we might want a position control.

Reply:
We can add it either now or later.